erpnext/projects/doctype/task/task.json

```diff
@@ -411,6 +410,7 @@
  ],
  "show_name_in_global_search": 1,
  "show_preview_popup": 1,
+ "show_title_field_in_link": 1,
```

Comment:
- I cannot see title field set in the doctype json

Reply:
![image](https://user-images.githubusercontent.com/60215482/119619976-2858b600-be22-11eb-86fd-19e2a1016e80.png)

@hrwX , Here the subject is set to the title field